src/main/java/emu/grasscutter/command/commands/ResetConstCommand.java

```diff
@@ -38,7 +38,6 @@ public void execute(Player sender, Player targetPlayer, List<String> args) {
 
     private void resetConstellation(Avatar avatar) {
         avatar.getTalentIdList().clear();
-        avatar.setCoreProudSkillLevel(0);
```

Comment:
`resetConstCommand` essentially does the same thing as `setConstCommand`, but with less features. Perhaps this command can be removed entirely.

Reply:
I agree.

Reply:
I could also add a `/cons reset` subcommand that would internally call `/cons set 0` if we want to.

Reply:
Only other regression would be "reset all"